### DISCUSSION TITLE
[Enhancemant] Add "disable/enable <Packagename>" option to subcommand "package"

### DISCUSSION THREAD

**Discussion by OntoDomina** [espanso/espanso#1220] [Feature requests and Ideas]
While Espanso greatly smooth my work, I found it inconvenient when using it within an editor, sometimes feeling necessary to switch between package. 
Also, this feature would also be useful for one who has a number of packages but doesn't want to use some of them for the time being.

Perhaps it could be as such
```
espanso package enable <PackageName>
espanso package disable <PackageName>
```

**Comment by federico-terzi** [comment]
Hey @OntoDomina,

Thank you for the feedback! I think that once #594 and #462 are implemented, this will become possible, so stay tuned :)

Cheers :)
